cmd/algorand-indexer/daemon.go

```diff
@@ -174,8 +180,22 @@ func (bih *blockImporterHandler) HandleBlock(block *types.EncodedBlockCert) {
 		logger.Errorf("received block %d when expecting %d", block.Block.Round, bih.round+1)
 	}
 	_, err := bih.imp.ImportDecodedBlock(block)
-	maybeFail(err, "ImportDecodedBlock %d: %v", block.Block.Round, err)
```

Comment:
why not print `err`?

Reply:
see the updated definition of maybeFail in main.go, it adds the error to the log message in a structured way